src/Function.cpp

```diff
@@ -77,6 +77,35 @@ static long cur_func_idx;				// Index into FuncList that we are currently workin
 static bool param_first=true;			// Flag to track output of commas
 static int builtin_functions_cnt;
 
+BooleanFunctionAttribute::BooleanFunctionAttribute(int attr_prob, string func_attr)
+{
+	FuncAttrProb = attr_prob;
+	name = func_attr;
+}
+
+void
+BooleanFunctionAttribute::output_attr(std::ostream &out)
+{
+	if(rnd_flipcoin(FuncAttrProb))
+		out << " __attribute__((" << name << "))";
```

Comment:
Please make an operator in `FunctionAttribute` which will do the wrapping: `__attribute__((` and `))`.

Reply:
Does it mean overloading operator `<<` in `FunctionAttribute`?

Reply:
It's up to you, please do not ask about a small implementation details. I hope you'll come up with a nice solution.

Reply:
Sure. I will try.